src/core/core.scale.js

```diff
@@ -649,6 +649,24 @@ var Scale = Element.extend({
 		return +this.getRightValue(rawValue);
 	},
 
+	/**
+	 * @private
+	 */
+	_getDimension: function() {
```

Comment:
`_getDimension` sounds funny to me. I only ever see it as "dimensions". How about `_getDimensions`? That might also work better if we decide to add more attributes to the return object

Reply:
Done